modal/container_process.py

```diff
@@ -151,6 +152,14 @@ async def _handle_input(data: bytes, message_index: int):
             self.stdin.write(data)
             await self.stdin.drain()
 
+        async def _send_window_resize(rows: int, cols: int):
+            # resize sequence:
+            # - 2 bytes for the number of rows (big-endian)
+            # - 2 bytes for the number of columns (big-endian)
+            dims = struct.pack(">HH", rows, cols)
+            self.stdin.write(dims)
+            await self.stdin.drain(_terminal_size=(rows, cols))
```

Comment:
do we still need `self.stdin.write(dims)` here? Isn't it sufficient to just `.drain(_terminal_size=(...))`?